### ISSUE TITLE
Desktop randomly freezes when using nvidia card

### ISSUE THREAD

**Issue by VikramBombhi** [JackHack96/dell-xps-9570-ubuntu-respin#45]
## The issue
My laptop randomly freezes when I am using the Nvidia card. I think only my display gets frozen and my keyboard and mouse are still inputting because I can still control my keyboard backlight when this happens.

## Steps to reproduce
I am switching to the Nvidia card via `sudo prime-select nvidia` followed by rebooting my laptop. After rebooting my laptop will randomly freeze, sometimes after hours sometimes after minutes. It should be noted I have experienced this issue while gaming and also while web browsing (although I believe its more common while gaming).

## What iv tried
I have tried switching my Nvidia driver to version 390, 396, and 410 and all have the same issue.

## Possible fix
I have read that this is a common issue and might be related to the kernel version I am currently using (`4.15.0-38-generic`) so if anyone is not experiencing this issue and is using a different version let me know and I can try switching to it.

**Comment by JackHack96** [comment]
I'm going to upgrade my nVidia driver version today and I'll use it (though I don't game, but I heavily use the browser). BTW I'm using Linux 4.19

**Comment by VikramBombhi** [comment]
to be clear this issue persisted even with Nvidia driver version 396. I will try upgrading my kernel to 4.19 today and report back.

**Comment by VikramBombhi** [comment]
I have upgraded my kernel to 4.19.0 and I am still experiencing the issue with both Nvidia drivers 396 and 410. I will report back any fixes if I find any.

**Comment by VikramBombhi** [comment]
I just gave up and decided to go with a fresh install of Ubuntu. For others that are experiencing this issue I was able to track it down to the GPU falling off the bus(whatever that means). The exact error was found by reading though the contents of `journalctl` where I found the error `NVRM: Xid (PCI:0000:01:00): 79, GPU has fallen off the bus.`.

My setup was a fresh install of ubuntu 18.04 and then running the xps-tweaks script manually after install. I am now using the respined iso @JackHack96 created and everything seems to be working fine. I will close this issue for now and reopen if the problem reappears.
